crates/bevy_ui/src/ui_node.rs

```diff
@@ -347,6 +347,10 @@ impl Default for Style {
 #[derive(Copy, Clone, PartialEq, Eq, Debug, Serialize, Deserialize, Reflect)]
 #[reflect(PartialEq, Serialize, Deserialize)]
 pub enum AlignItems {
+    /// Items are packed toward the start of the axis
+    Start,
+    /// Items are packed toward the end of the axis
+    End,
     /// Items are aligned at the start
```

Comment:
Could do with some documentation on the difference between `Start`/`End` and `FlexStart`/`FlexEnd` (difference is that `FlexDirection` being `RowReverse` or `ColumnReverse` flips `FlexStart`/`FlexEnd` but doesn't affect `Start`/`End`)

Reply:
> Could do with some documentation on the difference between `Start`/`End` and `FlexStart`/`FlexEnd` (difference is that `FlexDirection` being `RowReverse` or `ColumnReverse` flips `FlexStart`/`FlexEnd` but doesn't affect `Start`/`End`)

Agreed, I'll change them.